Search/Adapter/ZendLuceneAdapter.php

```diff
@@ -26,16 +26,12 @@
  * https://github.com/zendframework/ZendSearch
  * http://framework.zend.com/manual/1.12/en/zend.search.lucene.html
  *   (docs for 1.2 version apply equally to 2.0)
- *
- * Note this adapter implements localization by creating an index for each
- * locale if a locale is specified.
  */
 class ZendLuceneAdapter implements AdapterInterface
 {
     const ID_FIELDNAME = '__id';
     const CLASS_TAG = '__class';
 
-    // TODO: This fields should be handled at a higher level
     const URL_FIELDNAME = '__url';
     const TITLE_FIELDNAME = '__title';
     const DESCRIPTION_FIELDNAME = '__description';
```

Comment:
They are adapter specific. For example an adapter might have a better way of namespacing properties than this. The adapters are under no obligation to use the same names, and I don't want to imply that they should.